ControlHubWebSocketControlSpec.md

```diff
@@ -2,31 +2,1081 @@
 
 ## Implementation Note
 
-We don't want to fight with a user Op Mode over control of the hardware; that would be a recipe for trouble. Instead, my idea is to work within the Op Mode system. I would add a built-in, hidden Manual Control Op Mode (MCOM) that would provide hardware access to the WebSocket API. Using the API would auto-start the MCOM, and the user could go back to normal operation by stopping the MCOM from the Driver Station, like any other Op Mode. If a different Op Mode is already running, an error will be reported via the manual control API, and the original Op Mode will keep running.
+We don't want to fight with a user Op Mode over control of the hardware; that would be a recipe for trouble. Instead, my idea is to work within the Op Mode system. I would add a built-in, hidden Manual Control Op Mode (MC) that would provide hardware access to the WebSocket API. Using the API would auto-start the MC, and the user could go back to normal operation by stopping the MC from the Driver Station, like any other Op Mode. If a different Op Mode is already running, an error will be reported via the manual control API, and the original Op Mode will keep running.
```

Comment:
The Manual Control Op Mode is still abbreviated MCOM, if it's abbreviated. MC is just the namespace name, and an abbreviation for the concept of Manual Control, which can apply to both the Manual Control [Web Socket] API and the Manual Control Op Mode.